fitting/run_modeling.py

```diff
@@ -0,0 +1,224 @@
+#### libraries
+import numpy as np
+# np.seterr(all='raise') # raise errors, don't just print warnings
+from numpy import *
+from numpy.linalg import eig
+from numpy.random import normal
+from scipy.optimize import curve_fit
+from random import random as rand
+from scipy import stats
+from scipy.stats import poisson
+from matplotlib import pyplot as plt
+import time
+#### local files
+from utilities.physconsts import *
+from utilities.rbconsts import *
+from utilities.rbensemble import RbEnsemble as ensemble
+
+
+
+def retention_at_t(t, T=None, base_retention=0.9):
+        """ Procedure for simulating a release ("drop") and recapture experiment
+            to deduce the temperature of actual atoms in such an experiment.
+
+            Based on code by Mark, with some corrections
+            'wx': waist
+            'Tdepth': FORT temperature depth
+            'T': atom temp
+            'tmax': max time in units us
+            'steps': number of FORT drop outs
+            'events': number of release-recapture events per data pt
+            'wy': optional waist for eliptical FORT
+        """
+        T = abs(T)
+        events = 20000
+        factor = 1
+        w0 = 2.5e-6 * factor  # [m]
+        Tdepth = 1.5e-3 * factor  # [K]
```

Comment:
wx, wy, and Tdepth should be added as keyword arguments. to avoid breaking your code, you can give them defaults wx=0.7e-6, wy=0.7e-6, Tdepth=1e-3. Then please update the docstring to be consistent with this.